Dockerfile

```diff
@@ -89,13 +91,16 @@ RUN mkdir eksctl && curl -L ${EKSCTL_SOURCE_URL} \
     rm eksctl_${EKSCTL_VERSION}.tar.gz
 
 WORKDIR /home/builder/eksctl/
-RUN go mod vendor
-RUN cp -p LICENSE /usr/share/licenses/eksctl && \
-    /usr/libexec/tools/bottlerocket-license-scan \
-      --clarify /src/clarify.toml \
-      --spdx-data /usr/libexec/tools/spdx-data \
-      --out-dir /usr/share/licenses/eksctl/vendor \
-      go-vendor ./vendor
+# TODO - restore this with a fix for https://github.com/bottlerocket-os/bottlerocket-test-system/issues/288
+# For reasons not yet understood, this can take an hour or more in certain environments. For now we need
```

Comment:
I wonder if we can try leveraging the go toolchain from the Bottlerocket SDK like we do for Bottlerocket proper, see:
https://github.com/bottlerocket-os/bottlerocket/blob/9749af9973e4494768c2fac563172c753b6bc011/Makefile.toml#L258-L268

Reply:
The `docker-go` tool can be found here: https://github.com/bottlerocket-os/bottlerocket/blob/develop/tools/docker-go

Reply:
Can we can add these comments to #288 and figure it out later?